doc/reference/rpc_command.rst

```diff
@@ -0,0 +1,1227 @@
+Legacy RPC Commands
+===================
+
+
+.. This document is generated via the `development:generate-documentation` command
+
+
+.. contents::
+   :depth: 2
+   :backlinks: none
+   :local:
+
+
+.. _StatusHandler:
+
+
+StatusHandler
+-------------
+
+
+.. _RpcCommand_status_type:
+
+
+``type``
+""""""""
+
+
+**Default**: ``"formatted"``
+
+
+.. _FileInfoHandler:
```

Comment:
this needs to be prefixed also ideally, `_Rpc_FileInfoHandler` or similar, but not super important.

Reply:
Opted for `_Rpc_Handler` in this case.